crates/driver/src/boundary/mempool.rs

```diff
@@ -30,18 +30,26 @@ use {
 
 #[derive(Debug, Clone)]
 pub struct Config {
-    pub additional_tip_percentage_of_max_fee: Option<f64>,
+    pub additional_tip_percentage: f64,
     pub max_additional_tip: Option<f64>,
     pub gas_price_cap: f64,
     pub target_confirm_time: std::time::Duration,
     pub max_confirm_time: std::time::Duration,
     pub retry_interval: std::time::Duration,
     pub account: ethcontract::Account,
-    pub high_risk_disabled: bool,
     pub eth: Ethereum,
     pub pool: GlobalTxPool,
 }
 
+#[derive(Debug, Clone, Copy)]
+pub enum HighRisk {
+    Enabled,
+    Disabled,
+}
+
+/// The mempool to use for publishing settlements onchain. The public mempool
+/// of an [`Ethereum`] node can be used, or one of the private mempools offered
+/// by various transaction relay services.
```

Comment:
```suggestion
/// by various private networks.
```

I think flashbots is typically referred to as a private network.